ColorStripe/Views/Devices/DevicesView.swift

```diff
@@ -5,25 +5,23 @@
 import SwiftUI
 import CoreBluetooth
 
-struct DevicesView: View {
+struct DevicesView<T: ManagerWrapper>: View {
```

Comment:
I prefer to use more descriptive names for generics
```suggestion
struct DevicesView<Manager: ManagerWrapper>: View {
```